sdk/storage/azure-storage-queue/tests/test_queue.py

```diff
@@ -813,6 +813,7 @@ def test_unicode_create_queue_unicode_name(self):
 
     @record
     def test_unicode_get_messages_unicode_data(self):
+        pytest.skip("Uncomment after msrest fix")
```

Comment:
Is it possible that we could make this skip conditional on Python 2.7?

Reply:
We could use pytest "skipif" - https://pytest.readthedocs.io/en/2.8.7/skipping.html

Reply:
done